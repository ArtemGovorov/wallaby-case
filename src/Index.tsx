const style = require<CSSModule>("./ErrorMessage.scss");

import * as Vue from "vue";
import {Component} from "vue-property-decorator";
import {CreateElement} from "vue/types";
import {VNode} from "vue/types/vnode";

import jsxHelper from "util/jsx-helper";

@Component
class Index extends Vue {
  public render(createElement: CreateElement): VNode | null {
    const h = jsxHelper(createElement);

    return (
      <div class={style.Index}>
        Hello, Wallaby
      </div>
    );
  }
}

export default Index;
