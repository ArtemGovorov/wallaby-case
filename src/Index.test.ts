import Index from ".";

describe("Index", () => {
  it("Should render Hello world without an exception", () => {
    const i = new Index();
    i.$mount();
    expect(i.$el).toMatchSnapshot();
  });
});
