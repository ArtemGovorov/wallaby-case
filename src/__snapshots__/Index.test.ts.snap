// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Should render Hello world without an exception 1`] = `
<div
  class="Index"
>
  Hello, Wallaby
</div>
`;
